server/common/cookie.go

```diff
@@ -90,7 +91,7 @@ func JoinCookies(key string, cookieList []*http.Cookie) (string, error) {
 		}
 		sb.WriteString(strings.Join(parts[1:], ":"))
 	} else {
-		return "", fmt.Errorf("invalid cookie for key %s", key)
+		return "", fmt.Errorf("%s:%s", InvalidCookieError, key)
```

Comment:
Use `fmt.Errorf("xxxxx %w", InvalidCookieError)`, so that `errors.As()` can be used for error checking.

Reply:
I want to include the "key" also........